lib/options.js

```diff
@@ -386,6 +386,20 @@ module.exports = {
         usage: ['CONVERSION'],
         supportedIn: [VERSION20, VERSION30, VERSION31],
         supportedModuleVersion: [MODULE_VERSION.V2, MODULE_VERSION.V1]
+      },
+      {
+        name: 'Select request body type',
+        id: 'preferredRequestBodyType',
+        type: 'enum',
+        default: 'x-www-form-urlencoded',
+        availableOptions: ['x-www-form-urlencoded', 'form-data', 'raw', 'first-listed'],
+        description: 'When there are multiple content-types defined in the request body of OpenAPI, the conversion ' +
+          'selects the preferred option content-type as request body.If "first-listed" is set, the first ' +
+          'content-type defined in the OpenAPI spec will be selected.',
+        external: true,
+        usage: ['CONVERSION'],
+        supportedIn: [VERSION20, VERSION30, VERSION31],
+        supportedModuleVersion: [MODULE_VERSION.V2, MODULE_VERSION.V1]
```

Comment:
We should keep only `[MODULE_VERSION.V2]` here. Since v1 will not support this option.

Reply:
Copy paste mistake, I'll correct that.